forc-pkg/src/pkg.rs

```diff
@@ -62,7 +62,16 @@ pub enum DepKind {
     /// The dependency is a library and declared under `[dependencies]`.
     Library,
     /// The dependency is a contract and declared under `[contract-dependencies]`.
-    Contract,
+    Contract { salt: fuel_tx::Salt },
+}
+
+impl fmt::Display for DepKind {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            DepKind::Library => write!(f, "library"),
+            DepKind::Contract { salt: _ } => write!(f, "contract"),
```

Comment:
Should we include the salt in this? I don't quite remember where this `Display` impl for `DepKind` is used :thinking:

Reply:
This is only used for displaying errors when trying to depend on a program with invalid type so I explicitly left salt out to not cause ultra verbose errors. 

https://github.com/FuelLabs/sway/blob/master/forc-pkg/src/pkg.rs#L817